tensorflow_serving/example/inception_client.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 # ==============================================================================
 
-#!/usr/grte/v4/bin/python2.7
+#!/usr/bin/env python
```

Comment:
Did you mean to say
#!/usr/bin/env python2.7
in all of these files?

We should hopefully remove this line entirely soon, but gRPC currently doesn't support python3 so it would probably be nice to keep 2.7 as the default here.